hooks/post-command

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
 set -euo pipefail
 
-IMAGE="$BUILDKITE_PLUGIN_TRUFFLEHOG_IMAGE_URI"
-curl -sSfL -k https://raw.githubusercontent.com/trufflesecurity/trufflehog/main/scripts/install.sh | sh -s -- -b /usr/local/bin v3.81.9
-echo "Performing container scan"
-
-results = $(trufflehog docker --image="$IMAGE" --json --only-verified --fail)
-trufflehog_status=$?
-files = $(echo $results | jq '. | .SourceMetadata.Data.Docker.file')
+temp_results="/tmp/results.json"
```

Comment:
Note that there's no enforcement of required variables here.

Reply:
Added.